src/NServiceBus.Core/Serializers/XML/XmlDeserialization.cs

```diff
@@ -629,17 +629,21 @@ object GetPropertyValue(Type type, XmlNode n)
 
         PropertyInfo GetProperty(Type t, string name)
         {
-            IEnumerable<PropertyInfo> props;
-            cache.typeToProperties.TryGetValue(t, out props);
+            IEnumerable<PropertyInfo> properties;
+            if (!cache.typeToProperties.TryGetValue(t, out properties))
```

Comment:
Just realized that the only formatting change was https://github.com/Particular/NServiceBus/pull/4095/commits/91d44659b7956727f4288c07852501ff126e0612#diff-36144d5d3104efccfa5e06f871ca3cd8R280
Splitting that out into dedicated commit feels like overkill